package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "usb-detection",
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
Let's not bump the version in the PR. We can start a release once this gets into master.